grammar.js

```diff
@@ -177,29 +177,31 @@ module.exports = grammar({
     // Intended for table column names per `:help help-writing`.
     // TODO: children should be $.word (plaintext), not $.atom.
     column_heading: ($) => seq(
-      field('name', seq(choice($._atom_noli, $._uppercase_words), repeat($._atom))),
+      alias(seq(choice($._atom_noli, $._uppercase_words), repeat($._atom)), $.heading),
       '~',
       token.immediate('\n'),
     ),
 
     h1: ($) =>
-      seq(
-        token.immediate(field('delimiter', /============+[\t ]*\n/)),
-        repeat1($._atom),
+      prec(1, seq(
+        alias(token.immediate(field('delimiter', /============+[\t ]*\n/)), $.separator),
```

Comment:
why not call it `delimiter` since that is already the field name?
```suggestion
        alias(token.immediate(field('delimiter', /============+[\t ]*\n/)), $. delimiter),
```

Reply:
Good point. Do we need the field at all? This is only needed for scoping nodes (so you can capture all `(node)` that are in a specific `field`), but I don't think that's ever relevant since these are unique nodes?

Reply:
the fields were only nice to have. I personally don't use them

Reply:
Gone they are, then.

Reply:
I'll expose `'~'` as a `delimiter` then as well.